src/ontology/uberon-edit.obo

```diff
@@ -224725,6 +224725,14 @@ relationship: dc-contributor https://orcid.org/0009-0005-7919-4905
 relationship: part_of UBERON:0001043 ! esophagus
 property_value: dcterms-date "2024-11-14T15:00:21Z" xsd:dateTime
 
+[Term]
+id: UBERON:8800001
+name: thalamus
+def: "The thalamus is composed by two grey matter structure of the diencephalon one in each hemisphere. It is located in the center of the brain, above the mesencephalon and the midbrain. The thalamus is a relay center for sensory and motor informations. It is composed of 50–60 nuclei that project to specific cortical areas, each nucleus can receive inputs from multiple cortical regions, creating feedback loops that enhance relevant signals while suppressing irrelevant ones. The organisation of thalamic afferents and efferents is contralateral, influencing both sensory and motor functions." [PMID:12192499, PMID:26618760, PMID:28450543, PMID:31194341]
```

Comment:
Add species qualification on details - e.g. in primates it is compose of 50-60 nuclei...